submitter/submission.py

```diff
@@ -87,7 +95,7 @@ def from_message(cls, message):
             result_queue=result_queue,
         )
 
-    def create_item(self):
+    def create_item(self) -> dspace.item.Item:
```

Comment:
Optional: you could import `Item` directly as something like `DSpaceItem` from `dspace.item` to avoid the repetitiveness of `dspace.item.Item`

Reply:
I thought about that too... but I'm okay with it.  For something so vague like `Item`, I thought th context was actually more helpful than annoying.  But good eye.